flixel/FlxCamera.hx

```diff
@@ -1256,10 +1255,8 @@ class FlxCamera extends FlxBasic
 			{
 				return;
 			}
-			// This is temporal fix for camera's color
```

Comment:
I don't know, seemed very hacky to me, so I removed it. As far as I can tell everything is running fine.